src/app/containers/PageHandlers/withError.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { string } from 'prop-types';
+import ErrorMain from '../ErrorMain';
+
+const WithError = Component => {
+  return function WithErrorComponent({ error, ...props }) {
+    if (!error) return <Component {...props} />;
```

Comment:
I was gunna say i have a personal preference that we should flip this, but Im disagreeing with myself now :P